src/main/java/anab/Person/ManagePerson.java

```diff
@@ -2,10 +2,34 @@
 
 public class ManagePerson {
 
-    public static void main(String[] args) {
-        Person person = new Person();
+    static void printeazaText(String name) {
+        System.out.println("Hello!" + name);
     }
 
-}
+    public static void main(String[] args) {
+        Person personPerson = new Person("Ion", "Popescu", "Male", 15, "vasilica@gmail.com", false);
+
+        System.out.println("First Name:\t" + personPerson.getFirstName());
+        System.out.println("Last Name:\t" + personPerson.getLastName());
+        System.out.println();
+        String name = "Buru";
+        printeazaText("Ana" + name);
+        System.out.println();
+
+        personPerson.setGender("Male");
+        System.out.println("Gender type: " + personPerson.getGender());
+        System.out.println();
 
+        String newEmailAddress = "myEmail@google.com";
+        personPerson.setEmailAddress(newEmailAddress);
+        System.out.println();
+
+        personPerson.printPersonDetails();
+        System.out.println();
+
+      //  personPerson.setEmailRestricted();
```

Comment:
please remove also this commented line, if not needed anymore

Reply:
Commented line removed.